examples/etcd/resources/monitor-job.yaml

```diff
@@ -0,0 +1,48 @@
+apiVersion: appcontroller.k8s/v1alpha1
+kind: Definition
+metadata:
+  name: monitor-clustername-job
+meta:
+  timeout: 0
+job:
+  apiVersion: batch/v1
+  kind: Job
```

Comment:
AFAIU what you get from a Job is that you will know whether your pods terminated. The command here looks like it should be long-running pod.